step.yml

```diff
@@ -111,6 +111,23 @@ inputs:
         - "yes"
         - "no"
       is_required: true
+  - disable_index_while_building: "yes"
+    opts:
+      title: "Disable index while building"
+      description: |-
+        Disable indexing during the build.
+
+        Indexing is needed for
+        
+        * Autocomplete
+        * Ability to quickly jump to definition
+        * Get class and method help by alt clicking.
+
+        Which are not needed in CI environment.
```

Comment:
Please clarify that this is a build in xcode feature and not something in out step.

Reply:
Fixed in:
https://github.com/bitrise-steplib/steps-xcode-test/pull/111/commits/dbcded918e7354f29d6051ce30f3376994436422